src/components/container/IntradayHeartRateChart/index.ts

```diff
@@ -0,0 +1 @@
+export {default as IntradayHeartRateChart} from "./IntradayHeartRateChart";
```

Comment:
For consistency, this can just export `default` without the `as IntradayHeartRateChart`.  Then the `container/index.ts` can export `default as IntradayHeartRateChart`.